### PR TITLE
fix: restrict ht-filter-bar max height

### PR DESCRIPTION
**Pull request by arjunlalb**
## Description
Specify max height (equal to 5 rows) for advanced filter bar to avoid filter bar taking up all the available space when there are many filters in explorer page.

### Testing
Verified

### Checklist:
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: projects/components/src/filtering/filter-bar/filter-bar.component.scss [hunk 1]
```diff
@@ -6,11 +6,13 @@
   flex-direction: column;
   justify-content: center;
   min-height: 36px;
+  overflow-y: scroll;
   width: 100%;
   padding: 0 1px 0 8px;
   background: white;
   border: 1px solid $color-border;
   border-radius: 6px;
+  max-height: 180px;
```
Comment (itssharmasandeep) [inline projects/components/src/filtering/filter-bar/filter-bar.component.scss:15]: why and where this change is needed?

**Review by adisreyaj — APPROVED** [review]

File: projects/components/src/filtering/filter-bar/filter-bar.component.scss [hunk 2]
```diff
@@ -6,11 +6,13 @@
   flex-direction: column;
   justify-content: center;
   min-height: 36px;
+  overflow-y: scroll;
```
Comment (anandtiwary) [inline projects/components/src/filtering/filter-bar/filter-bar.component.scss:9]: This is debatable. Can we get ux team's opinion?

Comment (anandtiwary) [reply projects/components/src/filtering/filter-bar/filter-bar.component.scss:9]: Also, this is the advanced filter bar with the same name. It needs refactoring.

Comment (arjunlalb) [reply projects/components/src/filtering/filter-bar/filter-bar.component.scss:9]: @anandtiwary intention was to do it on the advance filter bar in the explorer page. the bug was filed for that.

File: projects/components/src/filtering/filter-bar/filter-bar.component.scss (hunk 1 above)
Comment (arjunlalb) [reply projects/components/src/filtering/filter-bar/filter-bar.component.scss:15]: description :)

**Pull request merged**